src/Sylius/Behat/Context/Api/Admin/LoginContext.php

```diff
@@ -139,6 +171,26 @@ public function iShouldBeNotifiedThatEmailResetInstructionHasBeenSent(): void
         Assert::same($this->client->getLastResponse()->getStatusCode(), Response::HTTP_ACCEPTED);
     }
 
+    /**
+     * @Then I should be notified that my password has been successfully changed
+     */
+    public function iShouldBeNotifiedThatMyPasswordHasBeenSuccessfullyChanged(): void
+    {
+        Assert::same($this->client->getLastResponse()->getStatusCode(), Response::HTTP_ACCEPTED);
+    }
+
+    /**
+     * @Then I should not be able to change my password again with the same token
+     */
+    public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
+    {
+        $this->client->executeCustomRequest($this->request);
+
+        Assert::same($this->client->getLastResponse()->getStatusCode(), Response::HTTP_INTERNAL_SERVER_ERROR);
```

Comment:
minor thing but it can be moved to variable `$this->client->getLastResponse()` as we use it twice